### ISSUE TITLE
GitHub Basics Workshop - Exercise #3 - Your 2nd Pull Request

### ISSUE THREAD

**Issue by CristinaRuth** [CristinaRuth/dev-together-madison#11]
## Your 2nd Pull Request

**_This exercise is a part of our [Github source control and basics workshop](https://github.com/CristinaRuth/dev-together-madison/blob/master/Learn/Workshops/Basics/GitHub). If you're completely new to Github and git, we highly recommend you read through the workshop contents if you want to learn both._** 

This exercise is to help you get familiar with the process on how to create a pull request against another user's repo.

**If you are not familiar with Git and GitHub and are completely new, please see #9 first before going through this exercise.**

In this exercise, you'll fork THIS repo and create a pull request. This mimics the process you'd be following for any Open Source contributions.

For terminologies, please refer to the [README.md](https://github.com/CristinaRuth/dev-together-madison/blob/master/Learn/Workshops/Basics/GitHub) in the workshop folder.

### Steps
1. Go to https://github.com/CristinaRuth/dev-together-madison and at the top right, click on fork. This should create a copy of the repo under your profile. 
1. Once the copy is complete, clone the repo to your machine and make changes - create a new `“sample-your-name-or-screenname.md”` file with any text within it under the `“Learn/Workshops/Basics/GitHub/Practice”` folder. See #9 and #10 for the steps and commands.
1. Go back to your copy of the repo on github.com, go to Pull Requests and create a new pull request. Notice how the comparison is between “base repository” and “head repository”. (similar to branches) Click on “create pull request”, put some descriptive text in the next screen and click on “Create pull request”
1. Your pull request is now pending! Once one of the collaborators of this repo approves and completes it, your commit will show in the base repository https://github.com/CristinaRuth/dev-together-madison.
